src/backend/storeManagers/gog/library.ts

```diff
@@ -676,7 +677,9 @@ export async function getMetaResponse(
         'If-None-Match': etag
       }
     : undefined
-  const metaUrl = buildData.data?.items[0]?.link
+  const metaUrl =
+    buildData.data?.items.find((build: BuildItem) => !build.branch)?.link ||
```

Comment:
```suggestion
    buildData.data?.items?.find((build: BuildItem) => !build.branch)?.link ||
```

do we need a `?` here? not sure, maybe we don't (I'm always confused what happens when chaining `?` operators)

Reply:
Maybe we should, I'm not sure how the response looks like when there are no builds available (that's the edge case but it doesn't hurt I guess.